node/node_syncing.go

```diff
@@ -103,12 +103,16 @@ func (node *Node) SupportSyncing() {
 	node.InitSyncingServer()
 	node.StartSyncingServer()
 	go node.SendNewBlockToUnsync()
-
 	if node.NodeConfig.Role() != nodeconfig.ShardLeader && node.NodeConfig.Role() != nodeconfig.BeaconLeader {
 		go node.DoSyncing(node.blockchain, node.Worker, node.GetSyncingPeers, true)
 	}
 }
 
+// GetSync gets sync-ed to blockchain without joining consensus
+func (node *Node) GetSync() {
+	go node.DoSyncing(node.blockchain, node.Worker, node.GetBeaconSyncingPeers, false) //Don't join consensus
```

Comment:
why redundant go routine call here?  You already called ```go GetSync```.

Reply:
sorry got copied over from a old commit.

Reply:
Actually I am NOT doing `go GetSync`. I fixed it correctly.